src/java.security.jgss/share/native/libj2gss/gssapi.h

```diff
@@ -45,9 +45,11 @@ extern "C" {
 
 // Condition was copied from
 // Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/gssapi/gssapi.h
```

Comment:
done

Reply:
This is rarely used in the code and is not the essence of the current changes.
If you introduce such changes, then throughout the code.
Moreover, this can lead to problems, such as, for example, here: https://bugs.openjdk.org/browse/JDK-8309225

Reply:
I think you can create a separate ticket and pull request to discuss this issue.